src/agent/include/process_options.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <string>
+
+void RegisterAgent(const std::string& user,
+                   const std::string& password,
+                   const std::string& key,
+                   const std::string& name);
+void StartAgent();
+void StatusAgent();
+void StopAgent();
```

Comment:
Could we add here the `restartAgent` function, to keep compatibility with windows development? The linux implementation can be left empty if it is not needed for now.

Reply:
Done. I moved the restart code to process_options_unix.cpp. That way we stick to a more consistent pattern.